auth/src/main/java/org/aerogear/android/ags/auth/AuthService.java

```diff
@@ -1,35 +1,30 @@
-package org.aerogear.auth;
+package org.aerogear.android.ags.auth;
 
-import org.aerogear.auth.credentials.ICredential;
-import org.aerogear.auth.impl.OIDCAuthCodeImpl;
-import org.aerogear.auth.impl.OIDCTokenAuthenticatorImpl;
+import android.content.Context;
+
+import net.openid.appauth.AuthState;
+
+import org.aerogear.android.ags.auth.credentials.ICredential;
+import org.aerogear.android.ags.auth.impl.OIDCAuthCodeImpl;
+import org.aerogear.android.ags.auth.impl.OIDCTokenAuthenticatorImpl;
+import org.aerogear.mobile.core.MobileCore;
+import org.aerogear.mobile.core.ServiceModule;
+import org.aerogear.mobile.core.configuration.ServiceConfiguration;
 
 import java.security.Principal;
 import java.util.concurrent.Future;
 
 /**
  * Entry point for authenticating users.
  */
-public class AuthService {
-
-    /**
-     * Authentication service singleton.
-     */
-    private static AuthService INSTANCE;
+public class AuthService implements ServiceModule {
```

Comment:
I presume this won't compile because its not implementing the methods from `ServiceModule` right?

Also how is the AuthService object got now?

Reply:
It is. Well CircleCI tests are passing. Jenkins is failing because the gradle daemon decided to go ahead and stop.

The ServiceModule interface has changed in this PR https://github.com/aerogear/aerogear-android-sdk/pull/22

Reply:
Ah right cool, thanks for explaining